fortuna/src/Login.js

```diff
@@ -1,49 +1,108 @@
 //@flow strict
 import * as React from 'react';
+import { Link } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import ReactDOM from 'react-dom';
-import MarketPlace from './MarketPlace.js';
+import LoginPopup from './LoginPopup.js';
 import './Login.css';
-  
+const PORT = 3000;
+
   //This is the Login Page Display
-  class LoginPageDisplay extends React.Component {
-    handleLoginClick = () => {
-        LoginFunction();
-      };
-    render(){
+  class Login extends React.Component<{||}> {
+    state = {
+      response: '',
+      userName: '',
+      password: '',
+      responseToPost: '',
+    };
+    
+    componentDidMount() {
+      this.callApi()
+        .then(res => this.setState({ response: res.express }))
+        .catch(err => console.log(err));
+    };
+    
+    callApi = async () => {
+      const response = await fetch('http://localhost:'+PORT+'/backend/routes/api/signup.js');
+      const body = await response.json();
+      if (response.status !== 200) throw Error(body.message);
+      
+      return body;
+    };
+    
+    handleLoginClick = async e => {
+      e.preventDefault();
+      const response = await fetch('http://localhost:'+PORT+'/backend/routes/api/signup.js', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ userName: this.state.userName, password:this.state.password }),
```

Comment:
You sure this passes flow? This seems super sketch since those things might not exist...